docs/architecture/high-performance-scala.md

```diff
@@ -1,8 +1,18 @@
 # JVM Primitives
 ------
```

Comment:
The `#` and `---` combination creates some funky double-underlining when rendered by github. Either `##` or `#` alone for these should work.